common/app/experiments/Experiments.scala

```diff
@@ -10,6 +10,15 @@ object ActiveExperiments extends ExperimentsDefinition {
   implicit val canCheckExperiment = new CanCheckExperiment(this)
 }
 
+object DCRJSBundleVariant
+    extends Experiment(
+      name = "dcr-js-bundle-variant",
+      description = "DCR bundle experiment",
+      owners = Seq(Owner.withGithub("guardian/dotcom-platform")),
+      sellByDate = never,
+      participationGroup = Perc0D,
```

Comment:
I think we can move to 1% already, as currently, the `variant` bunle is identical to the `modern` one.